libraries/libxml/source/org/pentaho/reporting/libraries/xmlns/writer/XmlWriterSupport.java

```diff
@@ -794,14 +803,19 @@ public void writeTextNormalized( final Writer writer, final String s, final bool
    * @param transformNewLine true, if a newline in the string should be converted into a character entity.
    * @return the normalised string.
    */
-  public String normalize( final String s, final boolean transformNewLine ) throws IOException {
+  public static String normalize( final String s, final boolean transformNewLine ) {
```

Comment:
Interestingly enough, there's a comment from Thomas on the previous commit warning that you should not change the accessibilty of the method